### ISSUE TITLE
ERROR: When trying to generate audio from text

### ISSUE THREAD

**Issue by jpvajda** [deepgram-starters/node-text-to-speech#2]
## Steps to reproduce 

* run npm start
*  Use the Main branch for testing
* select a voice 
* input text
* click play  button

## Error 

```bash
Server is listening on port 3000
Error writing audio to file: [Error: ENOENT: no such file or directory, open 'audio/audio.wav'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'audio/audio.wav'
}
[Error: ENOENT: no such file or directory, open 'audio/audio.wav'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'audio/audio.wav'
}
Error writing audio to file: [Error: ENOENT: no such file or directory, open 'audio/audio.wav'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'audio/audio.wav'
}
[Error: ENOENT: no such file or directory, open 'audio/audio.wav'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'audio/audio.wav'
}
```

**Comment by jpvajda** [comment]
@SandraRodgers 👋  This only seems to be an issue on Main, the audio ran fine on the `output-streaming` branch

**Comment by jpvajda** [comment]
I made the changes to fix this in #4 @SandraRodgers
